### PR TITLE
add disable strip tags support  [Check #9]  https://github.com/willvincent/feeds/pull/9

### PR DESCRIPTION
**Pull request by AhmedFawzy**
Some times we need to pass some thing like iframe so we need to apply this from the config file hope to see that alive soon .

Moved to a clean PR without the composer name changes.

https://github.com/willvincent/feeds/pull/9

### CODE REVIEW

**Comment by willvincent** [comment]
Please submit a clean PR without the composer name changes.

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "willvincent/feeds",
+    "name": "ahmedfawzy/feeds",
```
Comment (arthurkirkosa) [inline composer.json]: Why change the package name?

File: src/FeedsFactory.php [hunk 2]
```diff
@@ -19,6 +19,12 @@ public function make($feed_url) {
     $this->simplepie = new SimplePie();
     $this->configure();
     $this->simplepie->set_feed_url($feed_url);
+	if ( ! $this->config['strip_html_tags.disabled'] && ! empty( $this->config['strip_html_tags.tags'] ) && is_array($this->config['strip_html_tags.tags'])) {
```
Comment (arthurkirkosa) [inline src/FeedsFactory.php:22]: Can you extract this to a better named function so anyone can understand in one sentence what it does?

File: src/config/feeds.php [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+
+  /*
+  |--------------------------------------------------------------------------
+  | Striped Html Tags
+  |--------------------------------------------------------------------------
+  |
+  |
+  |
+  */
+//	'base', 'blink', 'body', 'doctype', 'embed', 'font', 'form', 'frame', 'frameset', 'html', 'iframe', 'input', 'marquee', 'meta', 'noscript', 'object', 'param', 'script', 'style'
```
Comment (arthurkirkosa) [inline src/config/feeds.php:22]: This can be moved to more lines for easier reading

**Comment by AhmedFawzy** [comment]
done
Please check
https://github.com/willvincent/feeds/pull/9

**Comment by willvincent** [comment]
Merged #9, thanks

**Pull request closed without merging**